<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImageClone</class>
 <widget class="QDialog" name="DialogImageClone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="le_targePath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>目标目录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_suffix">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>图片后缀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始克隆</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tip">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请开始克隆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>生成二维数组</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptn_addMirrorImg">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加镜像图片</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_quilt">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>功能选择</string>
   </property>
   <widget class="QRadioButton" name="rbn_rev">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>镜像</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbn_fetch">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>提取</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbn_reduce">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>降低画质</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
